README.md

```diff
@@ -84,7 +84,7 @@ The installation prerequisites are the same as described in the TensorFlow
 
         git clone https://github.com/NervanaSystems/ngraph-tf.git
         cd ngraph-tf
-        git checkout v0.12.0-rc1
+        git checkout v0.12.0-rc2
```

Comment:
@sayantan-nervana have the README change already here.

Reply:
Silly me 🤕